tiles/src/main/java/com/protomaps/basemap/layers/Water.java

```diff
@@ -37,8 +37,6 @@ public void processPreparedOsm(SourceFeature ignoredSf, FeatureCollector feature
   public void processNe(SourceFeature sf, FeatureCollector features) {
     var sourceLayer = sf.getSourceLayer();
     var kind = "";
-    var alkaline = 0;
-    var reservoir = 0;
```

Comment:
Those are unused variables

Reply:
These properties are in tilezen, should we keep them for now and add them to the output vector tiles?

Reply:
I guess we should treat this as cleanup for now, and also investigate how the Tangram styles ever used the alkaline/reservoir properties, because if they were never symbolized we don't need them...

Reply:
It is true though that alkaline was in the tilzen tiles, for ne see:

https://github.com/tilezen/vector-datasource/blob/89eeec6e98c6f5ae25d72970953ba93bb8b635a1/yaml/water.yaml#L250-L256

and for osm see:

https://github.com/tilezen/vector-datasource/blob/89eeec6e98c6f5ae25d72970953ba93bb8b635a1/yaml/water.yaml#L144-L147

Reply:
I suggest to remove the vector tile features properties `alkaline` and `reservoir`

Reply:
Removed also the OSM reservoir / alkaline

Reply:
no results for https://github.com/search?q=org%3Atangrams%20alkaline&type=code or reservoir=true in styles, so we can drop it